code/src/App.js

```diff
@@ -1,9 +1,83 @@
-import React from 'react'
+import React, {useState, useEffect} from 'react'
+
+import { Fetch_API, like_url } from './reusable/urls'
+
+import { CardContainer } from './components/CardContainer'
+import { Form } from './components/Form'
 
 export const App = () => {
+  const [thoughtsList, setThoughtsList] = useState([])
+  const [thoughtsNew, setThoughtsNew] = useState('')
+  const [loader, setLoader] = useState (true)  
+
+  useEffect(() => {
+    fetchThoughtsList()
+  }, [thoughtsList])
+
+  const fetchThoughtsList = () => {
+    fetch(Fetch_API)
+    .then(res => res.json())
+    .then(thoughts => setThoughtsList(thoughts))
+    .then(setLoader(false))
+    .catch(err => console.error(err))          
```

Comment:
nice adding an err catch 👍

Reply:
but you can't use console outputs to handle them. They can't be present in the production code.